content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -9,11 +9,18 @@ weight: 50
 ---
 
 <!-- overview -->
-A network policy is a specification of how groups of {{< glossary_tooltip text="pods" term_id="pod">}} are allowed to communicate with each other and other network endpoints.
 
-NetworkPolicy resources use {{< glossary_tooltip text="labels" term_id="label">}} to select pods and define rules which specify what traffic is allowed to the selected pods.
+If you want to control traffic flow at the IP address or port level (OSI layer 3 or 4), then you might consider using Kubernetes NetworkPolicies for particular applications in your cluster.  NetworkPolicies are an application-centric construct which allow you to specify how a {{< glossary_tooltip text="Pods" term_id="pod">}} is allowed to communicate with various network "entities" (we use the word "entity" here to avoid overloading the more common terms such as "endpoints" and "services", which have specific Kubernetes connotations) over the network. 
```

Comment:
@jayunit100 Jay, Pod here should be singular, right? PTAL at this: https://deploy-preview-23066--kubernetes-io-master-staging.netlify.app/docs/concepts/services-networking/network-policies/

Or maybe, as Network Policy is defined by Label, turn the whole phrase into plural: how some Pods are allowed to communicate with [...]